app/views/map/_leaflet.html.erb

```diff
@@ -1,8 +1,10 @@
-<script>
-        var map = L.map('map').setView([<%= lat %>,<%= lon %>], 10);
-        L.tileLayer("https://a.tiles.mapbox.com/v3/jywarren.map-lmrwb2em/{z}/{x}/{y}.png").addTo(map)
+<div id="map">
```

Comment:
I think we have to remove the map div from the sidebar templates too if we're going to add it here, or there'll be two maps in the sidebar. Make sense?